worksheet2/initLB.c

```diff
@@ -33,8 +34,50 @@ int readParameters(
     return 0;
 }
 
+void initialiseCell(double *collideField, double *streamField, int *flagField, int length_tot, int x, int y, int z, int flag){	
+
+	flagField[z * length_tot * length_tot + y * length_tot + x] = flag;
+
+	for (int i = 0; i < 19; ++i){
```

Comment:
By the way, sublime text also has to have an option, that tabs will be automatically changed to an equivalent number of spaces:)